pkg/filesystem/path/resolve_test.go

```diff
@@ -16,74 +16,74 @@ func TestResolve(t *testing.T) {
 	ctrl := gomock.NewController(t)
 
 	t.Run("NullByte", func(t *testing.T) {
-		scopeWalker := mock.NewMockScopeWalker(ctrl)
-
+		_, err := path.NewUNIXParser("hello\x00world")
```

Comment:
Technically speaking this test should be moved into unix_parser_test.go or something, but I have to say I don't care strongly.